install/gcp/gcp_installer.py

```diff
@@ -69,6 +69,9 @@ def run():
                         choices=['client', 'server'],
                         help='Type of the installation, '
                              'either client or server')
+    parser.add_argument('--target-project-id',
+                        help='The target project id, '
+                             'instead of the organization id.')
```

Comment:
please mention this is the project that is inventoried. without this flag, the entire org will get inventoried